queries.sql

```diff
@@ -8,3 +8,9 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IN ('1');
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+SELECT COUNT(*) FROM animals;
```

Comment:
- [ ] Good job so far 👍 There are some transactions that you have to carry out in the queries.sql before answering the queries. It would be great if you could add those 😄 
- Requirements:
![Annotation 2022-05-07 211402](https://user-images.githubusercontent.com/85113567/167270371-b2eba082-6a67-4a52-9d7f-097a8cfbac21.jpg)

Reply:
Roger that, working on it. Thnks!